extensions/podman/src/podman-cleanup-macos.ts

```diff
@@ -0,0 +1,58 @@
+import type { ProviderCleanupExecuteOptions } from '@podman-desktop/api';
+import { process } from '@podman-desktop/api';
+import psList from 'ps-list';
+import { homedir } from 'node:os';
+import { resolve as pathResolve } from 'node:path';
+import { AbsPodmanCleanup } from './podman-cleanup-abstract';
+
+// Handle cleanup of Podman on macOS
+export class PodmanCleanupMacOS extends AbsPodmanCleanup {
+  async killProcess(processId: number): Promise<void> {
+    await process.exec('/bin/kill', ['-9', String(processId)]);
```

Comment:
Would be better to use SIGTERM or SIGSTOP instead of SIGKILL as it allow cleanup

Reply:
does it really matter as we do the cleanup ?

Reply:
I also think that it would be better to try to use SIGTERM/SIGSTOP first, and use SIGKILL after a grace period. The process may want to cleanup some temporary resources it has created, that we don't know about

Reply:
I'm not sure you meant SIGSTOP ?

Reply:
Effectively, SIGSTOP doesn't seem appropriate. SIGTERM first, then SIGKILL later.

Reply:
added usage of SIGTERM instead of SIGKILL which I still believe is a bad idea